src/Umbraco.Web.UI/umbraco/config/lang/da.xml

```diff
@@ -1004,17 +1005,91 @@ Mange hilsner fra Umbraco robotten
     <key alias="preview">Vis prøve</key>
     <key alias="styles">Styles</key>
   </area>
+
   <area alias="template">
-    <key alias="edittemplate">Rediger skabelon</key>
+    <key alias="edittemplate">Rediger skabelong</key>
+
+    <key alias="insertSections">Sektioner</key>
     <key alias="insertContentArea">Indsæt indholdsområde</key>
-    <key alias="insertContentAreaPlaceHolder">Indsæt indholdsområdemarkering</key>
-    <key alias="insertDictionaryItem">Indsæt ordbogselement</key>
-    <key alias="insertMacro">Indsæt makro</key>
-    <key alias="insertPageField">Indsæt Umbraco sidefelt</key>
+    <key alias="insertContentAreaPlaceHolder">Indsæt indholdsområde placeholder</key>
+
+    <key alias="insert">Indsæt</key>
+    <key alias="insertDesc">Hvad vil du indsætte ?</key>
+
+    <key alias="insertDictionaryItem">Oversættelse</key>
+    <key alias="insertDictionaryItemDesc">Indsætter en oversætbar tekst som skifter efter det sprog som websitet vises i.</key>
+
+    <key alias="insertMacro">Makro</key>
+    <key alias="insertMacroDesc">
+      En makro er et element som kan have forskellige indstillinger når det indsættes.
+      Brug det som en genbrugelig del af dit design såsom gallerier, formularer
+      og lister.
+    </key>
+
+    <key alias="insertPageField">Side værdi</key>
+    <key alias="insertPageFieldDesc">
+      Viser værdien af et felt fra den nuværende side. Kan indstilles til at bruge rekursive værdier eller
+      vise en standard værdi i tilfælde af at feltet er tomt.
+    </key>
+
+    <key alias="insertPartialView">Partial view</key>
+    <key alias="insertPartialViewDesc">
+      Et Partial View er et skabelon element som kan indsættes i andre skabeloner og derved
+      genbruges og deles på tværs af side-skabelonerne.

```

Comment:
Et Partial View er et **skabelonelement**, som kan indsættes i andre skabeloner og derved genbruges og deles på tværs af **sideskabelonerne**. 
Skabelonelement and sideskabelonerne in one word, added a comma